pkg/asset/installconfig/platformprovisioncheck.go

```diff
@@ -70,8 +71,15 @@ func (a *PlatformProvisionCheck) Generate(dependencies asset.Parents) error {
 		if err != nil {
 			return err
 		}
-	case aws.Name, libvirt.Name, none.Name, openstack.Name, ovirt.Name:
+	case aws.Name, libvirt.Name, none.Name, openstack.Name:
 		// no special provisioning requirements to check
+
+	case ovirt.Name:
+		err = icovirt.ValidateProvisioning(ic.Config)
+		if err != nil {
+			return err
+		}
+
```

Comment:
move this up, lets leave the no-work and default at the end.

Reply:
done